examples/README.md

```diff
@@ -0,0 +1,25 @@
+# Sedna-examples
+This repository to share Sedna examples and demonstrate Sedna can implement across edge-cloud collaborative training 

```

Comment:
repository => directory?

Reply:
repository => directory, done it
"to share" is ok